source/openpulse/openpulse/ast.py

```diff
@@ -0,0 +1,85 @@
+"""
+
+========================================
+Abstract Syntax Tree (``openpulse.ast``)
+========================================
+
+.. currentmodule:: openpulse.ast
+
+The reference abstract syntax tree (AST) for OpenPulse programs.
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import List, Optional
+
+# Re-export the existing AST classes from openqasm3
+# pylint: disable=unused-import
+from openqasm3.ast import *
```

Comment:
We import everything from openqasm3.ast to reuse, but override `CalibrationDefinition`.